topostats/filters.py

```diff
@@ -255,11 +255,12 @@ def gaussian_filter(self, image: np.ndarray, **kwargs) -> np.array:
 
         """
         LOGGER.info(
-            f"[{self.filename}] : Applying Gaussian filter (mode : {self.gaussian_mode}; Gaussian blur (nm) : {self.gaussian_size})."
+            f"[{self.filename}] : Applying Gaussian filter (mode : {self.gaussian_mode}; Gaussian blur (px) : {self.gaussian_size})."
         )
+        print(f"Sigma = {self.gaussian_size / self.pixel_to_nm_scaling}")
```

Comment:
Erroneous `print()` should be `LOGGER.info()` if its meant to be output during processing, otherwise remove.